### PR TITLE
Removed markdown from COPYING

### PR DESCRIPTION
**Pull request by jackgavigan**
Updated the Copyright statement at the top of the file from "2016" to "2016-2017" and removed markdown markup from the main text.

### CODE REVIEW

File: COPYING [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2016 The Zcash developers
+Copyright (c) 2016-2017 The Zcash developers
 Copyright (c) 2009-2015 The Bitcoin Core developers
```
Comment (daira) [inline COPYING]: The Bitcoin copyright date should also be updated, because we've merged stuff from upstream.

Comment (jackgavigan) [inline COPYING]: 👍

**Comment by bitcartel** [comment]
ACK

**Comment by bitcartel** [comment]
@zkbot r+

**Pull request merged**